lib/command/command.js

```diff
@@ -65,7 +65,16 @@ externals.Commands = class {
   validate (slackResponse) {
     return Promise.resolve({
       then: (onFulfill, onReject) => {
-        if (!internals.isCommandAllowed(this.getCommand(),
+        if (!internals.isAllowedChannel(this.getCommand(), slackResponse,
+            this.getSlackData())) {
+          /* jshint ignore:start */
+          return onReject({
+            allowed_channels: true,
```

Comment:
Hi usubram ,

Thanks for your review !!.

These access privileges are bound to be on the bot level. When I mean allowed channels, bots are allowed only to those channels. But the  restricted channels key means to me in opposite way that  is the bot is restricted to those channels and allowed to all other channels 

does this make sense to you?